docs/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -267,15 +277,18 @@ If a CometBFT network selects a `MessageDelay` parameter that does not accuratel
 A very common way to update `ConsensusParams` is by executing a transaction included in a block that specifies new values for them.
 However, if the network is unable to produce blocks because of this liveness issue, no such transaction may be executed.
 To prevent this dangerous condition, we will add a relaxation mechanism to the `Timely` predicate.
+
+<!---
```

Comment:
Can we describe here the decision we make on Mon? Or is it coming in a further PR?

Reply:
Not sure. I would wait for the implementation. Moreover, we have to document the chosen solution, so that PR could also update this ADR.

Reply:
OK